README.md

```diff
@@ -49,27 +49,67 @@ The following utilities must be present for installation purposes:
 * docker
 * docker-compose
 
-# Installation 
+# Installation
 
 This project is designed to be portable. We use docker to ensure that there is
 clear separation between our tool's runtime environment and the
 rest of your system.
 
-To install, use the following command: 
+To clone, use the following command:
 
+```bash
+git clone https://github.com/ctrando/UCSD-Planner-Helper
+cd UCSD-Planner-Helper
 ```
-git clone https://github.com/ctrando/UCSD-Planner-Helper && cd UCSD-Planner-Helper 
+
+Run this app using a simple shell script:
+
+* To download the data, add "-d" flag. Note the scraping might take a while depends on your network environment.
+* The app can be either run in development mode or production mode. Production mode uses Nginx.
+* If any changes made to the files (e.g. update), add "-b" flag to tell docker to rebuild. Otherwise docker will always use the images existing.
+* "-s" flag is used to stop the detached services in production mode.
+
+```bash
+./scripts/run.sh [-h|--help] [-d|--download] [-p|--production] [-b|--build] [-s|--stop] [-c|--cert <email>]
+```
+
+Usage:
+
+```bash
+./scripts/run.sh -h        # display help message
+
+./scripts/run.sh           # run in development mode without reloading or rebuilding
+./scripts/run.sh -d        # run in development mode and download the data
+./scripts/run.sh -p        # run the app in production mode
+./scripts/run.sh -p -d     # run the app in production mode, and download data
+./scripts/run.sh -b -p -d  # rebuild, run the app in production mode, and download data
+
+./scripts/run.sh -s        # stop the running production mode services
+                           # give error if production services not started properly
+./scripts/run.sh -c e@mail # FIRST PRODUCTION RUN ONLY
```

Comment:
type

Reply:
typo*

Reply:
That is intended to tell people input an email address

Reply:
Sorry, you have a typo in -c e@mail